gettext.go

```diff
@@ -36,70 +40,80 @@ func DefaultResolver(root string, locale string, domain string) string {
 	return path.Join(root, locale, "LC_MESSAGES", fmt.Sprintf("%s.mo", domain))
 }
 
-// NewTranslations is the main entry point for gogettext. Use this to set up
-// the locales for your app.
-// root is the root of your locale folder, domain the domain you want to load
-// and resolver a function that resolves mo file paths.
-// If your structure is <root>/<locale>/LC_MESSAGES/<domain>.mo, you can use
-// DefaultResolver.
-func NewTranslations(root string, domain string, resolver PathResolver) Translations {
-	return Translations{&translations{
-		root:     root,
-		resolver: resolver,
-		domain:   domain,
-		cache:    map[string]Catalog{},
-	}}
+// Translations is a deprecated alias for a pointer to a TextDomain struct
+type Translations = *TextDomain
+
+// NewTranslations initialises a TextDomain struct, setting the Name,
+// LocaleDir and PathResolver fields.
```

Comment:
Should this also get a comment that it's deprecated? If not it feels slightly strange to have "NewFoo" that returns Bar :)

Reply:
I've marked this method as deprecated, and used the form suggested by the Go developers (a paragraph in the docstring beginning with "Deprecated:").